l1-contracts/contracts/state-transition/chain-deps/DiamondInit.sol

```diff
@@ -66,6 +66,13 @@ contract DiamondInit is ZkSyncHyperchainBase, IDiamondInit {
         s.feeParams = _initializeData.feeParams;
         s.blobVersionedHashRetriever = _initializeData.blobVersionedHashRetriever;
 
+        if (_initializeData.batchCommitDeadline == 0) {
+            s.batchCommitDeadline = 3 days;
+        } else {
+            s.batchCommitDeadline = _initializeData.batchCommitDeadline;
+        }
```

Comment:
```suggestion
        s.batchCommitDeadline = _initializeData.batchCommitDeadline;
```

1) No implicit conversion. Why 3 days? Why can't set zero timestamp? 
2) We should keep the parameter to be a part of configuration so testnet STM can set a different `_initializeData` but reuse the same contracts.